package.json

```diff
@@ -32,14 +32,18 @@
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@ng-bootstrap/ng-bootstrap": "^10.0.0",
     "@tiptap/core": "^2.0.0-beta.204",
+    "@tiptap/extension-code-block-lowlight": "^2.0.0-beta.209",
```

Comment:
This version should match the other TipTap extensions at 2.0.0-beta.204 if possible